apps/src/templates/feedback/LevelFeedback.jsx

```diff
@@ -0,0 +1,98 @@
+import React, {Component} from 'react';
+import i18n from '@cdo/locale';
+import color from '@cdo/apps/util/color';
+import {levelFeedbackType} from './types';
+import LevelFeedbackEntry from '@cdo/apps/templates/feedback/LevelFeedbackEntry';
+import Button from '@cdo/apps/templates/Button';
+
+export default class LevelFeedback extends Component {
```

Comment:
This is totally fine, but just a reminder that functional components with hooks are also an option now!

Reply:
HOOKS! I forgot, this seems like a good opportunity to start using them, I'll convert.